File: station/src/main/java/com/example/model/Vehicle.java
package com.example.model;

import lombok.Getter;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Getter
@Setter
@Entity
public class Vehicle {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private String  name;
    private String vehicleType;
    private String fromPoint;
    private String endPoint;
    private String phone;
    private String email;
    private String startTime;
    private String endTime;
}

File: vehicle/src/app/model/vehicle.ts
import {PositionPr} from "./position-pr";

export interface Vehicle {
  id?:number;
  name?:string;
  vehicleType?:string;
  fromPoint?: string;
  endPoint?:string;
  phone?:string;
  email?:string;
  startTime?:string;
  endTime?:string;
}
